app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -593,3 +579,4 @@ function AcuantCapture(
 }
 
 export default forwardRef(AcuantCapture);
+export { AcuantDocumentType };
```

Comment:
I'm confused about why we now export `AcuantDocumentType` from both `acuant-capture` and `acuant-camera`.  Is it possible to remove this line?

Reply:
That makes sense! Thanks for the explanation.